src/components/LoginApollo.js

```diff
@@ -39,9 +39,9 @@ const Login = ({ googleUser, setGoogleUser, clientID, location, history }) => {
     };
   };
 
-  const handleLogin = googleUserLogin => {
+  const handleLogin = async googleUserLogin => {
     setGoogleUser(googleUserLogin);
-    login();
+    await login();
```

Comment:
Or just leave as is

Reply:
supposedly if there are any errors, we should read from `error` returned by `useMutation` instead

Reply:
The point I am trying to make is that `handleLogin` need not be `async` since we are not returning a `Promise`.

And since we are not returning anything, there is no need to await for `login()`.

Reply:
I see